view/theme/frio/js/textedit.js

```diff
@@ -2,6 +2,55 @@
  * @brief The file contains functions for text editing and commenting
  */
 
+function initComment(callback) {
```

Comment:
What is the purpose of this function?

Reply:
I used the `function initEditor(callback)` from `jot-header.tpl` as a guide and removed not related lines. 
If I'm seeing it now autocomplete would be a nice thing too.
The callback line remained because I couldn't tell if really needed.

Reply:
Just replace 
```javascript
initComment(function(){
    addcommenttext(data, id);
});
```
by
```javascript
addcommenttext(data, id);
```
And remove `initComment()` entirely.